packages/components/src/components/SkeletonText/index.tsx

```diff
@@ -11,7 +11,7 @@ const pulse = keyframes`
 
 export const SkeletonText = styled(Element)(props => {
   const color = props.theme.colors.sideBar.border;
-  const themeType = props.theme.type;
+  const themeType = props.theme.type || props.theme.vscodeTheme.type;
```

Comment:
hate doing this, but this based on if you use <ThemeProvider> or `withTheme` inside the component.

`withTheme` use cases would vanish over time, but right there are a few 🙃

Reply:
why this so sad